packages/typespec-ts/src/modular/buildClassicalClient.ts

```diff
@@ -370,3 +389,108 @@ function addChildClient(
     )`
   );
 }
+
+function hasTenantLevelOperations(
+  client: SdkClientType<SdkServiceOperation>,
+  dpgContext: SdkContext
+): boolean {
+  const methodMap = getMethodHierarchiesMap(dpgContext, client);
+
+  for (const [_, operations] of methodMap) {
+    for (const op of operations) {
+      // Check if this operation has a client-level subscriptionId parameter
+      const hasSubscriptionIdParam = op.operation.parameters?.some(
+        (param) =>
+          param.name.toLowerCase() === "subscriptionid" &&
+          param.kind === "path" &&
+          param.onClient
+      );
+
+      if (!hasSubscriptionIdParam) {
+        // Skip certain operation types that are not relevant for tenant-level detection
+        const pathLC = op.operation.path.toLowerCase();
+        const clientNamespaceLC = client.namespace.toLowerCase();
+        if (
+          pathLC.includes(`${clientNamespaceLC}/operations`) ||
+          pathLC.includes(`${clientNamespaceLC}/locations`) ||
+          pathLC.includes(`${clientNamespaceLC}/registeredsubscriptions`) ||
+          pathLC.includes(`${clientNamespaceLC}/checknameavailability`)
```

Comment:
why these 4 are excluded? any reason we use path to compare not operation name + namesapce?